sos/report/plugins/charmed_postgresql.py

```diff
@@ -0,0 +1,191 @@
+# This file is part of the sos project: https://github.com/sosreport/sos
+#
+# This copyrighted material is made available to anyone wishing to use,
+# modify, copy, or redistribute it subject to the terms and conditions of
+# version 2 of the GNU General Public License.
+#
+# See the LICENSE file in the source distribution for further information.
+
+import os
+import yaml
+
+from sos.report.plugins import Plugin, UbuntuPlugin
```

Comment:
It seems `charmed-postgresql` is available on many distributions, per https://snapcraft.io/charmed-postgresql, but since "Users by distribution" show just Ubuntu, we can have just `UbuntuPlugin` until somebody wants otherwise.